lib/scripts.js

```diff
@@ -63,10 +62,11 @@ module.exports = function (gulp, gutil) {
         ]
       }))
       .pipe(wrapAngularContents())
-      .pipe(ngAnnotate()({
-        single_quotes: true
-      }))
       .pipe(gutil.env.filterFiles(files))
+        .pipe(ngAnnotate()({
+          single_quotes: true
+        }))
+       .pipe(uglify()({compress: true, mangle: true}))
```

Comment:
Do not compress until specified from argument passed on command line. You can access this using `guilt.env` as we have elsewhere.